src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -74,31 +76,42 @@ private JObject GetInvokeResult(byte[] script, Cosigners cosigners = null)
             {
                 json["stack"] = "error: recursive reference";
             }
-            ProcessInvokeWithWallet(json, cosigners);
+            ProcessInvokeWithWallet(json, signers);
             return json;
         }
 
+        private static Signers SignersFromJson(JArray _params)
+        {
+            return new Signers(_params.Select(u => new Signer()
```

Comment:
We can serialize it and deserialize it in order to check that all the information it's good

https://github.com/neo-project/neo/blob/54784e03091aa217e133fbe3e70288d36c985e58/src/neo/Network/P2P/Payloads/Signer.cs#L28-L39

Reply:
But it's not convenient for the caller.

Reply:
If he fill a wrong signer, he's tx will never arrive

Reply:
@shargon can you help modify this😅